service/pom.xml

```diff
@@ -22,14 +22,15 @@
         <tds.session.client>0.0.1-SNAPSHOT</tds.session.client>
         <tds.common>0.0.1-SNAPSHOT</tds.common>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <java.version>1.8</java.version>
         <unit-tests.skip>false</unit-tests.skip>
         <integration-tests.skip>true</integration-tests.skip>
         <docker.image.prefix>sbacoss</docker.image.prefix>
         <assertj.version>3.4.1</assertj.version>
         <mockito.version>1.10.19</mockito.version>
         <rest-assured.version>2.9.0</rest-assured.version>
         <h2-database.version>1.4.191</h2-database.version>
+        <java.version>1.8</java.version>
+        <maven.compiler.plugin.version>3.5.1</maven.compiler.plugin.version>
```

Comment:
We should settle on what we want to do here.  Do we want to put all versions up in this section or not?  I admit I wasn't consistent when I initially configured these services.  I don't feel strongly, but my main reason for having versions up in the property section was if things were duplicated.  For example `java.version` is referenced in a couple places so it is up here.  Does assertj need to be up here?  

Like I said I'm fine with all versions being up here but lets make sure we do that in all projects.